pkg/operators/eventgen/params.go

```diff
@@ -0,0 +1,58 @@
+package eventgen
+
+import (
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/gadget-service/api"
+)
+
+const (
+	name                  = "eventgen"
+	ParamEventGenEnable   = "eventgen-enable"
+	ParamEventGenType     = "eventgen-type"
+	ParamEventGenTarget   = "eventgen-target"
+	ParamEventGenCount    = "eventgen-count"
+	ParamEventGenInterval = "eventgen-interval"
+)
+
+const (
+	EventTypeDNS  = "dns"
+	EventTypeHTTP = "http"
+)
+
+// getGlobalParams returns the operator's global parameters
+func getGlobalParams() api.Params {
+	return api.Params{
```

Comment:
@flyth @blanquicet It seems the global param doesn't work for `ig-k8s`. I assume if we want an operator to be client side but have a global param right now it doesn't work right now? I mean server has no idea about this operator?